### DISCUSSION TITLE
Adding external converter for custom nRF device

### DISCUSSION THREAD

**Discussion by gsapinto** [Koenkk/zigbee2mqtt#8174] [Devices]
Hello there! I'm trying to integrate a device I've made with nRF52840 with the Z2M dongle. This custom nrf device was developed with custom clusters in order to expose 2 temperatures and an AC current value. I've already paired the device but I am unable to detect any of the readings it is sending over zigbee in the Z2M dongle.
I'm having trouble in adding a custom external converter to the z2m. I've already found a "template" for these custom converters here: https://gist.github.com/nurikk/b9c7389e0abf81fec9a1b99baa21f49c
I've tried creating my own but I don't know what I'm missing or what I'm doing wrong. 
Here's mine so far:
[generic_dryer.txt](https://github.com/Koenkk/zigbee2mqtt/files/6901930/generic_dryer.txt)
What I think I'm missing is a way to identify the cluster I want to use, i.e its ID, but I'm not sure where to do that.
Any help would be appreciated! @Koenkk

**Comment by avdept** [comment]
Hey, @gsapinto could you share info how did you make Z2M to see your NRF52840 DK? I uploaded NRF's bulb zigbee example - but Z2M does not see it at all.
